README.md

```diff
@@ -26,16 +38,32 @@ cd wallabag && make install
 
 Now, [configure a virtual host](https://doc.wallabag.org/en/admin/installation/virtualhosts.html) to use your wallabag.
 
-# Run on YunoHost
+### Other methods
+
+#### Docker
+
+You can also use the official Docker image, available at https://hub.docker.com/r/wallabag/wallabag
+
+#### YunoHost
```

Comment:
Shouldn't we redirect to the documentation about installation instead of listing other methods to install it?
https://doc.wallabag.org/en/admin/installation/installation.html#installation-with-docker

Because otherwise, we only _showcase_ Docker & YunoHost installation, and we are hiding Cloudron, Alwaysdata, etc.

Reply:
50f36363286d94670c5dacabc57d81e38c6769c1